modules/integration/tests-integration/tests-mediator-1/src/test/java/org/wso2/carbon/esb/mediator/test/payload/factory/FormatPayloadWithOMTypeCTXExpressionTestCase.java

```diff
@@ -0,0 +1,88 @@
+/*
+*Copyright (c) 2005-2010, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*WSO2 Inc. licenses this file to you under the Apache License,
+*Version 2.0 (the "License"); you may not use this file except
+*in compliance with the License.
+*You may obtain a copy of the License at
+*
+*http://www.apache.org/licenses/LICENSE-2.0
+*
+*Unless required by applicable law or agreed to in writing,
+*software distributed under the License is distributed on an
+*"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+*KIND, either express or implied.  See the License for the
+*specific language governing permissions and limitations
+*under the License.
+*/
+
+package org.wso2.carbon.esb.mediator.test.payload.factory;
+
+import org.apache.axiom.om.OMAbstractFactory;
+import org.apache.axiom.om.OMElement;
+import org.apache.axiom.om.OMFactory;
+import org.apache.axiom.om.OMNamespace;
+import org.apache.axis2.AxisFault;
+import org.apache.axis2.Constants;
+import org.apache.axis2.client.Options;
+import org.apache.axis2.client.ServiceClient;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.esb.integration.common.utils.ESBIntegrationTest;
+
+import static org.testng.Assert.assertFalse;
+
+public class FormatPayloadWithOMTypeCTXExpressionTestCase extends ESBIntegrationTest {
+	@BeforeClass(alwaysRun = true)
+	public void uploadSynapseConfig() throws Exception {
+		super.init();
+		loadESBConfigurationFromClasspath("/artifacts/ESB/mediatorconfig/payload/factory/om_ctx_payload_factory_synapse.xml");
```

Comment:
Better to use [File.separator](http://docs.oracle.com/javase/7/docs/api/java/io/File.html#separator) so the test can be run in other OS(windows)